.dockerignore

```diff
@@ -1,9 +1,16 @@
-.vagrant/
-_build/
-apps/aecuckoo/c_src
-apps/aecuckoo/priv/bin
-apps/aecuckoo/priv/lib
-apps/aecore/priv/keys
-system_test
-erl_crash.dump
-aevm_external
+*
+!rebar.config
+!rebar.lock
+!Makefile
+!rebar3
+!rebar.config.script
+!VERSION
+!apps
```

Comment:
Some files in this folder are actually derived, that may defeat the whitelist.

Most of such files/folders are cleaned in https://github.com/aeternity/epoch/blob/d10630530df4254d925bd6784854f1fef358b429/Makefile#L278-L282